example/pokeapi_functional/lib/main.dart

```diff
@@ -13,14 +13,8 @@ class MyApp extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     /// [TextEditingController] using hooks
     final controller = useTextEditingController();
-    final requestStatus = ref.watch(pokemonProvider);
-    useEffect(() {
-      /// Fetch the initial pokemon information (random pokemon).
-      Future.delayed(Duration.zero, () {
-        ref.read(pokemonProvider.notifier).fetchRandom();
```

Comment:
Is this request done somewhere else with this refactoring?

Reply:
In the Riverpod v2's Notifier, initialization occurs during the build method